dask_sql/context.py

```diff
@@ -20,10 +20,16 @@
     LogicalPlan,
 )
 
+try:
+    from dask_sql.physical.utils.statistics import parquet_statistics
+except ModuleNotFoundError:
+    parquet_statistics = None
+
```

Comment:
Are there cases where we wouldn't be able to import?

Reply:
I added this because our bare requirements test errored with:
```
import pyarrow.parquet as pq
ModuleNotFoundError: No module named 'pyarrow'
```

Reply:
This was with my first commit to this PR https://github.com/dask-contrib/dask-sql/actions/runs/4129811432/jobs/7135835860

Reply:
Ah I see. I think this is okay for now then. cc: @charlesbluca if you have thoughts